modules/st2-menu/menu.component.js

```diff
@@ -118,8 +118,8 @@ export default class Menu extends React.Component {
           Docs
         </a>
 
-        <a href="mailto:support@stackstorm.com" className="st2-menu__side" data-reamaze-lightbox="contact">
```

Comment:
How this change affects `bwc-ui`?

I agree that it's good idea to remove support email for Enterprise version, but I think it makes sense to leave it in Enterprise UI.